aldryn_config.py

```diff
@@ -329,4 +329,11 @@ def to_settings(self, data, settings):
         # This may need to be removed in a future release.
         settings['CMS_UNESCAPED_RENDER_MODEL_TAGS'] = data['unescaped_render_model_tags']
 
+        # Prevent injecting random comments to counter BREACH/CRIME attacks
+        # into the page tree snippets, as the javascript parsing the result
+        # expects a single top-level element.
```

Comment:
truth is it expects all top level nodes to be valid pagetree entries, there can be multiple, they just have to be valid list items and not comments

but yeah, whatever 🎉